lib/active_merchant/billing/gateways/pay_gate_xml.rb

```diff
@@ -251,6 +270,13 @@ def commit(action, request)
           :authorization  => response[:tid]
         )
       end
+      def commit_capture(action, authorization, request)
```

Comment:
Another note. Only reason that function was split up like that is because the refundtx request requires the tid from the authrx response and not the capturerx response. So i had to make sure that the purchase function returns the authrx's tid instead of the capturerx because the responses tid are infact not the same at paygate